src/AutoMapper/QueryableExtensions/ProjectionExpression.cs

```diff
@@ -87,7 +87,8 @@ public IQueryable<TResult> To<TResult>(IObjectDictionary parameters, params Expr
 
         internal IQueryable<TResult> To<TResult>(IObjectDictionary parameters, MemberPaths memberPathsToExpand)
         {
-            var membersToExpand = memberPathsToExpand.SelectMany(m => m).Distinct().ToArray();
+            IEnumerable<MemberInfo> selectMany = memberPathsToExpand.SelectMany(m => (m ?? new List<MemberInfo>()) as IList<MemberInfo> ?? m /*?.ToList() */);
+            MemberInfo[] membersToExpand = selectMany.Distinct().ToArray();
```

Comment:
m should never be null. So the right behavior here is to crash, not check for null. That only makes it more difficult to debug.